src/ota.cpp

```diff
@@ -46,9 +46,12 @@ const char* u_host = "217.9.195.227";
 const char* u_url = "/esp/firmware.php";
 const char* firmware_update_path = "/upload";
 
+extern const char *esp_hostname;
+
 void ota_setup()
 {
   // Start local OTA update server
+  ArduinoOTA.setHostname(esp_hostname);
```

Comment:
This probably is a result of the re-order I did, but ArduinoOTA calls MDNS.begin and resets the hostname, this change just ensures that no matter what order they are called in the same host name is set.